### PR TITLE
added GeeksForGeeks

### PR DESCRIPTION
**Pull request by arin17bishwa**
added https://www.geeksforgeeks.org/ , a coding platform

### CODE REVIEW

File: wmn-data.json [hunk 1]
```diff
@@ -2004,6 +2004,16 @@
         "known" : ["tommy", "cderalow"],
         "cat" : "health"
        },
+        {
+        "name" : "GeeksForGeeks",
+        "uri_check" : "https://www.geeksforgeeks.org/user/{account}",
```
Comment (WebBreacher) [inline wmn-data.json:2009]: When I visit this URL it redirects me to `https://www.geeksforgeeks.org/user/{account}/`. Please add the trailing `/`.

File: wmn-data.json [hunk 2]
```diff
@@ -2004,6 +2004,16 @@
         "known" : ["tommy", "cderalow"],
         "cat" : "health"
        },
+        {
+        "name" : "GeeksForGeeks",
+        "uri_check" : "https://www.geeksforgeeks.org/user/{account}",
+        "e_code" : 200,
+        "e_string" : "|  GeeksforGeeks Profile</title>",
+        "m_code" : 404,
```
Comment (WebBreacher) [inline wmn-data.json:2012]: I get a 307 code that redirects my browser to the 404.html page.

File: wmn-data.json [hunk 3]
```diff
@@ -2004,6 +2004,16 @@
         "known" : ["tommy", "cderalow"],
         "cat" : "health"
        },
+        {
+        "name" : "GeeksForGeeks",
+        "uri_check" : "https://www.geeksforgeeks.org/user/{account}",
+        "e_code" : 200,
+        "e_string" : "|  GeeksforGeeks Profile</title>",
+        "m_code" : 404,
+        "m_string" : "Go to Homepage</a>",
```
Comment (WebBreacher) [inline wmn-data.json:2013]: This `m_string` is too generic and may appear on other pages too. Please consider changing to `<title>Page Not Found - `

**Comment by WebBreacher** [comment]
Thanks for the submission! A few things to adjust before I can add this to the project. Please see my comments above.

File: wmn-data.json (hunk 2 above)
Comment (arin17bishwa) [reply wmn-data.json:2012]: I used Python requests and tried to get an invalid username, and it was returned with 404 as the final response code. If I do `response.history` it does give me the intermediate 307. `wget` has the same behaviour. 
I am not sure which status code is actually expected.


image

Comment (WebBreacher) [reply wmn-data.json:2012]: For the project, we use the first response code (307) instead of the final one.

**Comment by WebBreacher** [comment]
Thanks for the submission!

**Pull request merged**